com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl

```diff
@@ -18,31 +18,18 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Version.hlsl"
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl"
 
-#if !defined(SHADER_HINT_NICE_QUALITY)
-    #if defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH)
-        #define SHADER_HINT_NICE_QUALITY 0
-    #else
-        #define SHADER_HINT_NICE_QUALITY 1
-    #endif
-#endif
-
-// Shader Quality Tiers in Universal.
-// SRP doesn't use Graphics Settings Quality Tiers.
-// We should expose shader quality tiers in the pipeline asset.
-// Meanwhile, it's forced to be:
-// High Quality: Non-mobile platforms or shader explicit defined SHADER_HINT_NICE_QUALITY
-// Medium: Mobile aside from GLES2
-// Low: GLES2
-#if SHADER_HINT_NICE_QUALITY
-    #define SHADER_QUALITY_HIGH
-#elif defined(SHADER_API_GLES)
-    #define SHADER_QUALITY_LOW
-#else
-    #define SHADER_QUALITY_MEDIUM
+// Opt-in lower shader quality.
+//#define _SHADER_QUALITY_LOW 1
+#ifndef _SHADER_QUALITY_LOW
+#define _SHADER_QUALITY_HIGH 1
```

Comment:
I wonder if it makes sense to remove this now, most users won't know about this and we plan in short term to add shader quality tiers anyway.

Reply:
We can remove it. I decided to leave it in, so you can still opt-in by defining SHADER_QUALITY_LOW explicitly.
This way we know/remember what the previous optimizations were and if someone relied on them they can still get them.
But more code is more maintenance too.

Reply:
Removed.